MzTab_MTD

```diff
@@ -94,67 +94,54 @@ CREATE TABLE "MzTab"."Metadata_Proteomics"
     "colunit-peptide" text COLLATE pg_catalog."default",
     "colunit-psm" text COLLATE pg_catalog."default",
     "colunit-small_molecule" text COLLATE pg_catalog."default",
+    "Internal_mzTab-ID" integer NOT NULL DEFAULT nextval('"MzTab"."Metadata_Proteomics_Internal_ID_seq"'::regclass),
     CONSTRAINT "Metadata_Proteomics_pkey" PRIMARY KEY ("mzTab-ID")
 )
 
 TABLESPACE pg_default;
 
 ALTER TABLE "MzTab"."Metadata_Proteomics"
     OWNER to postgres;
-    
-    
--- Table: MzTab.Protein section
+-- Table: MzTab.PSM Section
 
--- DROP TABLE "MzTab"."Protein section";
+-- DROP TABLE "MzTab"."PSM Section";
 
-CREATE TABLE "MzTab"."Protein section"
+CREATE TABLE "MzTab"."PSM Section"
 (
+    sequence text COLLATE pg_catalog."default" NOT NULL,
+    "PSM_ID" integer NOT NULL,
```

Comment:
Is this supposed to be an integer?  I am thinking it should a sequence, correct?